### DISCUSSION TITLE
The spatial reference identifier (SRID) is not valid.

### DISCUSSION THREAD

**Discussion by robotrick** [ChilliCream/hotchocolate#5249] [General]
`query {
  cities (
    where: {
      location: {
        within: { geometry: { type: Point, coordinates: [-73.935242, 40.730610], lt: 250 }}
      }
    }) 
  
  {id location {type coordinates}}
}`

always results in error message "the spatial reference identifier (SRID) is not valid", thrown by MS SQL Server 2012. the provided sql filter value is 0x00000000010CB610E4A0845D44401FF64201DB7B52C0.

the query without "where" clause works.

using HC.spartial 12.11.1 with microsoft.entityframeworkcore.sqlserver.nettopologysuite 6.0.3
                
Any idea why this happens?

**Comment by PascalSenn** [comment]
What is the SRID the data is stored in?

**Reply by robotrick** [reply]
According to MS SQL Server, the binary is malformed and the SRID is unknown. It also cannot be converted.
